content/plotly/concepts/graph-objects/terms/scatterpolar/scatterpolar.md

```diff
@@ -0,0 +1,61 @@
+---
+Title: '.Scatterpolar()'
+Description: 'Creates a scatter plot on polar axes.'
+Subjects:
+  - 'Data Science'
+  - 'Data Visualization'
+Tags:
+  - 'Plotly'
+  - 'Graphs'
+  - 'Data'
+  - 'Values'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The **`.Scatterpolar()`** function in Plotly is used to create scatter plots on polar axes. Scatter plots are used to visualize data points in a two-dimensional space, and polar axes are circular axes that are often used to represent periodic data. 
+
+## Syntax
+
+```pseudo
+plotly.graph_objects.Scatterpolar(x=None, y=None, ...)
+```           
+
+- `x`: A list of values to be used as the x-coordinates of the data points.  
+- `y`: A list of values to be used as the y-coordinates of the data points.  
+
+> Note: The ellipsis in the syntax (...) indicates that there are additional optional parameters beyond those listed here to customize the scatter plot on polar axes.   
```

Comment:
```suggestion
> **Note:** The ellipsis in the syntax (...) indicates that there are additional optional parameters beyond those listed here to customize the scatter plot on polar axes.   
```
```suggestion
> **Note:** The ellipsis in the syntax (...) indicates that there are additional optional parameters beyond those listed here to customize the scatter plot on polar axes.   
```
The note word must be mentioned in bold when working with Codecademy docs. Also, I think the lines are redundant here so remove one of them for more clarity.